app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,8 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :credentials,                  :class_name => "ManageIQ::Providers::AutomationManager::Authentication",
```

Comment:
Long term:  I'd like to see two different tables and models (naming is hard, so I don't really care about the specific names used in my list below):

1. `Authentications` for storing authentication data used by MIQ for authenticating on the users' behalf
2. `Credentials` for storing provider inventory records representing credentials that are used by the provider in transactions.

These `AutomationManger` authentication records fall into the second category.

I don't really care which name we use now for this association.  But, I want there to be a clear path forward for distinguishing these types of things in the future so we can refactor it later.